### DISCUSSION TITLE
Voodoo 2 drivers

### DISCUSSION THREAD

**Discussion by Alevam-Inc** [86Box/86Box#3087] [Q&A] [answered]
When I try to install the drivers for the 3dfx Voodoo 2 on my VM, Windows 98 says that it already has the correct drivers installed. I had the Voodoo 2 enabled when I installed Win98SE so is this true?

**Comment by Flo9696** [comment, marked as answer]
Try start a game that supports Voodoo 2, and see if you can select Voodoo 2 as graphic card in the game.

Or else can you download the drivers from:
http://vogonsdrivers.com/index.php?catid=12
